rocky/tools/forms/base.py

```diff
@@ -68,6 +68,14 @@ class ObservedAtForm(BaseRockyForm):
         help_text=OBSERVED_AT_HELP_TEXT,
     )
 
+    def clean_observed_at(self):
+        observed_at = self.cleaned_data["observed_at"]
+        now = datetime.datetime.now(tz=datetime.timezone.utc).date()
+        diff = observed_at - now
+        if diff.days > 0:
```

Comment:
There's no need for an intermediate diff variable, you can just compare the two dates